gapic-generator-cloud/templates/cloud/helpers/method_helper.rb

```diff
@@ -0,0 +1,97 @@
+# frozen_string_literal: true
+
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+require "active_support/inflector"
+
+module MethodHelper
+  def method_name method
```

Comment:
hm I'm trying to wrap my head around these helpers. Crazy thought, what if we monkey patched the `Google::Gapic::Schema::{Element}` with these methods rather than prepending each helper with the element name.

Reply:
The schema structure is not a 1 to 1 for the ruby code structure. I am extremely hesitant to bolt all these ruby code structure methods onto the schema objects. IMO they need to remain separate until we have a firm grasp on what features are needed and how they will be implemented.

Think of the helper methods as well named placeholders for describing the ruby code needs.